_opensearch/rest-api/tasks.md

```diff
@@ -142,6 +143,60 @@ GET /_tasks?nodes=opensearch-node1
 }
 ```
 
+The following request will return details about active search tasks:
```

Comment:
Maybe also include that this will return detailed information?

Reply:
Done